PlayCover/Utils/SaveGenshinUserData.swift

```diff
@@ -56,13 +60,12 @@ func storeUserData( folderName: String, accountRegion: String ) {
 
             try fileManager.copyItem(at: lastAccountModelEncryptUrl,
                                      to: URL(fileURLWithPath: storePath + lastAccountModelEncrypt))
-            fileManager.createFile(atPath: storePath + "region.txt", contents: nil, attributes: nil)
-            try accountRegion.write(to: URL(fileURLWithPath: storePath + "region.txt"),
-                                    atomically: false, encoding: .utf8)
-
-        } catch {
-            Log.shared.error("Error moving file: \(error)")
-        }
+            if BUNDLE_ID == "com.miHoYo.GenshinImpact" {
```

Comment:
You're repeating this same check several times. Instead, make a bool to store this in order to avoid repeat string comparisons. Maybe pass this in to the function from PlayAppView so you have it from the very start.